zen_queries/render.py

```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render as django_render
+from zen_queries import queries_disabled
+
+
+def render(
```

Comment:
I think a better way of handling this would be to use a decorator, and pass in the existing `render` function, rather than duplicate all the args, kwargs, and default values.

Reply:
I thought of this approach first, but sadly the use of `contextlib.contextmanager` as a decorator was only added in Python 3.2: https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager (see last line of that docs section)

Reply:
Damn! Perhaps a more generic implementation then? You could easily create a `with_queries_disabled` decorator (which raises deprecation warnings in python3) and use it here?

Reply:
I think that's overcomplicating things. The context manager can already by used as a decorator if you're using Python 3.2+.

The main reason I duplicated the args (as opposed to just accepting `*args, **kwargs` and passing them to `django_render`) is so that `help(render)` still works as expected. I suppose I could do this and then put `render.__doc__ == django_render.__doc__` after the function, if you think that's better?

Reply:
I see. I think it's fine to have different docs for the function, seeing as they do different things (slightly). Perhaps adding a docstring to this function stating that it's a wrapper around `render` would suffice?

Reply:
OK